packages/core/src/Stores/client-store.js

```diff
@@ -813,6 +814,11 @@ export default class ClientStore extends BaseStore {
         return isMultipliersOnly(this.residence);
     }
 
+    @computed
+    get wallet_accounts() {
+        return this.active_accounts.filter(acc => acc.account_type === 'wallet');
```

Comment:
```suggestion
        return this.active_accounts.filter(account => account.account_type === 'wallet');
```
@iman-fs Avoid abbreviations.